Makefile

```diff
@@ -88,7 +88,7 @@ docs: .docker-build-pull
 .docker-build-ci:
 	${MAKE} build-ci
 
-build-ci: .env .docker-build-pull submodules
```

Comment:
I just noticed that these were superfluous since the `pull` step has them as dependencies.